signbank/static/js/gloss_edit.js

```diff
@@ -137,11 +137,13 @@ $.editable.addInputType('multicheckbox', {
             return submitdata;
         };
 
-        $.each(settings.data || [], function(value, label) {
+        // Output multicheckbox in alphabetical order of labels
+        const data_inverted = Object.fromEntries(Object.entries(settings.data).map(([k, v]) => [v, k]));
+        Object.keys(data_inverted).sort().forEach(function(label) {
+            var value = data_inverted[label];
```

Comment:
I left this as `var` because this is an old JS file and everything else is `var`, not `let` or `const`.

Reply:
that argument doesn't really hold when you've used `const` two lines above 😅

Reply:
D'oh - the habits!